cmd/k8s-operator/deploy/chart/values.yaml

```diff
@@ -39,6 +39,13 @@ operatorConfig:
 
   securityContext: {}
 
+  # ACL tag that operator will be tagged with. Operator must be made owner of
+  # these tags
+  # https://tailscale.com/kb/1236/kubernetes-operator/?q=operator#setting-up-the-kubernetes-operator
+  # Multiple tags can be passed as a comma-separated string i.e 'tag:k8s-operator,tag:prod'.
+  # Note that if you pass multiple tags to this field via `--set` flag to helm upgrade/install commands you must escape the comma (for example, "tag:k8s-operator\,tag:prod"). See https://github.com/helm/helm/issues/1556
+  defaultTags: "tag:k8s-operator"
```

Comment:
Could we please move this under `operatorConfig`?

Reply:
Additionally (thanks to https://github.com/Shaked for the tip), could we please make this into an array and then concatenate like `value: '{{ join "," .Values.operatorConfig.defaultTags }}'` in the deployment manifest? That way it would be easier for folks to pass the value via `--set`.

(The reason why it's _not_ done like that for the proxy tags is that we only discovered that it causes folks issues when passing multiple tags after releasing the chart with a string field)